gettingStarted/sysElevenStackKickstart.yaml

```diff
@@ -16,30 +16,44 @@ description: Simple template to deploy a single compute instance
 
 parameters:
  key_name:
-  type: string
+   type: string
+   default: 
```

Comment:
Why did you add a default? Without key_name the stack won't work.

Reply:
I forgot to remove that, used a default in the script before.